lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2022 PANTHEON.tech s.r.o. All Rights Reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+package io.lighty.applications.util;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.lighty.core.controller.impl.config.ConfigurationException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Objects;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ModulesConfig {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ModulesConfig.class);
+    private static final String MODULES_TIMEOUT_ELEMENT_NAME = "moduleTimeoutSeconds";
+    private static final long DEFAULT_TIMEOUT_SECONDS = 60;
+
+    private long moduleTimeoutSeconds = DEFAULT_TIMEOUT_SECONDS;
+
+    /**
+     * Load ModulesConfig configuration from InputStream containing JSON data.
+     *
+     * @param jsonConfigInputStream InputStream containing ModulesConfig configuration data in JSON format.
+     * @return Object representation of configuration data.
+     * @throws ConfigurationException In case InputStream does not contain valid JSON data,
+     *     or cannot bind Json tree to type.
+     */
+    public static ModulesConfig getModulesConfig(final InputStream jsonConfigInputStream)
+            throws ConfigurationException {
+        final ObjectMapper mapper = new ObjectMapper();
+        final JsonNode configNode;
+        try {
+            configNode = mapper.readTree(jsonConfigInputStream);
+        } catch (final IOException e) {
+            throw new ConfigurationException("Cannot deserialize Json content to Json tree nodes", e);
+        }
+        if (!configNode.has(MODULES_TIMEOUT_ELEMENT_NAME)) {
```

Comment:
IMO: just because we are missing one config field it does not mean that we have to use default configuration for all fields..